homeassistant/components/velux/const.py

```diff
@@ -0,0 +1,8 @@
+"""Constants for the Velux integration."""
+from logging import getLogger
+
+from homeassistant.const import Platform
+
+DOMAIN = "velux"
+PLATFORMS = [Platform.COVER, Platform.LIGHT, Platform.SCENE]
+_LOGGER = getLogger(__name__)
```

Comment:
package loggers should be public

```suggestion
LOGGER = getLogger(__package__)
```